app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,43 @@
+class InventoriesController < ApplicationController
+  def index
+    @inventory = Inventory.all
+  end
+
+  def show
+    @inventory = Inventory.find(params[:id])
+  end
+
+  def new
+    @inventory = Inventory.new
+  end
+
+  def create
+    @inventory = Inventory.new(inventory_params)
+    if @inventory.save
+      redirect_to inventory_path(param[:id])
+    else
```

Comment:
- [ ] After creating the inventory, I got the error message. Here, it should be **params** instead of param.

Reply:
Fixed🙂